anecdote/manifest.json

```diff
@@ -0,0 +1,67 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "35d64545-cee9-4eb6-98be-65cb9fdd944a",
+  "app_id": "anecdote",
+  "display_on_public_website": false,
+   "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor bugs reported by your customers in your customer feedback in your DataDog dashboard.",
+    "title": "Anecdote",
+    "media": [{
+        "media_type": "image",
+        "caption": "The Anecdote app provides a clear, near-real-time overview of bugs reported by your customers in your customer feedback.",
```

Comment:
Please include shorter descriptions of the image

Reply:
Fixed